### PR TITLE
Plotting HOM J

### PR DESCRIPTION
**Pull request by amylu00**
## Purpose 

Including plots for homogeneous nucleation rate coefficient to verify values.

## To-do

- compare J_hom to that in Koop 2000
- move plots to IceNucleation.md?
- uncomment @asserts in Common.jl (H2SO4 vapor pressure)

## Content

- plot of log10(J_hom) vs T where J is in cm^-3 s^-1

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by amylu00** [comment]
Calculating Delta_a_w the way Koop 2000 did at x = 0.032  and constant P = 0.01 GPa gives me
Screenshot 2023-07-12 at 3 49 30 PM

But the max valid P decreases each iteration and goes into the negatives.

**Comment by amylu00** [comment]
bors try

**Comment by amylu00** [comment]
For all plots, blue curve is our parameterization and yellow curve are paper values

x_sulph = 0.031 (minimum value that doesn't give NaN error)
x_0 31

x_sulph = 0.04
x_0 4

x_sulph = 0.06
x_0 6

File: docs/src/HomFreezingPlots.jl [hunk 1]
```diff
@@ -0,0 +1,138 @@
+import OrdinaryDiffEq as ODE
```
Comment (trontrytel) [inline docs/src/HomFreezingPlots.jl:1]: Where do we need the ODE solver here?

File: docs/src/HomFreezingPlots.jl [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+import CairoMakie as MK
+
+import Thermodynamics as TD
+import CloudMicrophysics as CM
+import CLIMAParameters as CP
+
+const CMT = CM.CommonTypes
+const CMO = CM.Common
+const CMI_het = CM.HetIceNucleation
```
Comment (trontrytel) [inline docs/src/HomFreezingPlots.jl:10]: This one doesn't seem to be used

File: docs/src/HomFreezingPlots.jl [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+temp = collect(229.0:0.25:239.5)   # air temperature
+#x_sulph = 0.031                    # wt% sulphuric acid in droplets
+#x_sulph = 0.04
+x_sulph = 0.065
+Delta_a = Vector{Float64}(undef, length(temp))
+J = Vector{Float64}(undef, length(temp))
+a_w = Vector{Float64}(undef, length(temp))
+
+it = 1
+for T in temp
```
Comment (trontrytel) [inline docs/src/HomFreezingPlots.jl:30]: Could I ask you to update this one to the same way of plotting as the water activity ones - i.e. without the loop and the global it?

File: docs/src/HomFreezingPlots.jl [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    p_sol = CMO.H2SO4_soln_saturation_vapor_pressure(x_sulph, T)
+    #p_sat = TD.saturation_vapor_pressure(thermo_params, T, TD.Liquid())
+
+    p_ice = exp(9.550426 - 5723.265 / T + 3.53068 * log(T) - 0.00728332 * T)
+    ln_p_sat =
+        54.842763 - 6763.22 / T - 4.210 * log(T) +
+        0.000367 * T +
+        tanh(0.0415 * (T - 218.8)) *
+        (53.878 - 1331.22 / T - 9.44523 * log(T) + 0.014025 * T)
+    p_sat = exp(ln_p_sat)
```
Comment (trontrytel) [inline docs/src/HomFreezingPlots.jl:43]: Could we be using the p_ice from Thermodynamics?

**Review by trontrytel — APPROVED** [review]

**Pull request merged**